src/features/pastelExpertConsole/rpc-services/utils.ts

```diff
@@ -0,0 +1,979 @@
+
+export const METHODS = [
+  // Addressindex
+  'getaddressbalance',
+  'getaddressdeltas',
+  'getaddressmempool',
+  'getaddresstxids',
+  'getaddressutxos',
+
+  // Blockchain
+  'getbestblockhash',
+  'getblock',
+  'getblockchaininfo',
+  'getblockcount',
+  'getblockdeltas',
+  'getblockhash',
+  'getblockhashes',
+  'getblockheader',
+  'getchaintips',
+  'getdifficulty',
+  'getmempoolinfo',
+  'getrawmempool',
+  'getspentinfo',
+  'gettxout',
+  'gettxoutproof',
+  'gettxoutsetinfo',
+  'verifychain',
+  'verifytxoutproof',
+  'z_gettreestate',
+
+  // control
+  'getexperimentalfeatures',
+  'getinfo',
+  'getmemoryinfo',
+  'help',
+  'setlogfilter',
+  'stop',
+
+  // disclosure
+  'z_getpaymentdisclosure',
+  'z_validatepaymentdisclosure',
+
+  // Generating
+  'generate',
+  'getgenerate',
+  'setgenerate',
+
+  // Mining
+  'getblocksubsidy',
+  'getblocktemplate',
+  'getlocalsolps',
+  'getmininginfo',
+  'getnetworkhashps',
+  'getnetworksolps',
+  'prioritisetransaction',
+  'submitblock',
+
+  // Network
+  'addnode',
+  'clearbanned',
+  'disconnectnode',
+  'getaddednodeinfo',
+  'getconnectioncount',
+  'getdeprecationinfo',
+  'getnettotals',
+  'getnetworkinfo',
+  'getpeerinfo',
+  'listbanned',
+  'ping',
+  'setban',
+
+  // Rawtransactions
+  'createrawtransaction',
+  'decoderawtransaction',
+  'decodescript',
+  'fundrawtransaction',
+  'getrawtransaction',
+  'sendrawtransaction',
+  'signrawtransaction',
+
+  // Util
+  'createmultisig',
+  'estimatefee',
+  'estimatepriority',
+  'validateaddress',
+  'verifymessage',
+  'z_validateaddress',
+
+  // Wallet
+  'addmultisigaddress',
+  'backupwallet',
+  'dumpprivkey',
+  'dumpwallet',
+  'encryptwallet',
+  'getaccount',
+  'getaccountaddress',
+  'getaddressesbyaccount',
+  'getbalance',
+  'getnewaddress',
+  'getrawchangeaddress',
+  'getreceivedbyaccount',
+  'getreceivedbyaddress',
+  'gettransaction',
+  'getunconfirmedbalance',
+  'getwalletinfo',
+  'importaddress',
+  'importprivkey',
+  'importpubkey',
+  'importwallet',
+  'keypoolrefill',
+  'listaccounts',
+  'listaddressgroupings',
+  'listlockunspent',
+  'listreceivedbyaccount',
+  'listreceivedbyaddress',
+  'listsinceblock',
+  'listtransactions',
+  'listunspent',
+  'lockunspent',
+  'move',
+  'sendfrom',
+  'sendmany',
+  'sendtoaddress',
+  'setaccount',
+  'settxfee',
+  'signmessage',
+  'z_exportkey',
+  'z_exportviewingkey',
+  'z_exportwallet',
+  'z_getbalance',
+  'z_getmigrationstatus',
+  'z_getnewaddress',
+  'z_getnotescount',
+  'z_getoperationresult',
+  'z_getoperationstatus',
+  'z_gettotalbalance',
+  'z_importkey',
+  'z_importviewingkey',
+  'z_importwallet',
+  'z_listaddresses',
+  'z_listoperationids',
+  'z_listreceivedbyaddress',
+  'z_listunspent',
+  'z_mergetoaddress',
+  'z_sendmany',
+  'z_setmigration',
+  'z_shieldcoinbase',
+  'z_viewtransaction',
+  'zcbenchmark',
+  'zcrawjoinsplit',
+  'zcrawkeygen',
+  'zcrawreceive',
+  'zcsamplejoinsplit',
+]
```

Comment:
This allows to use this as a type
```suggestion
] as const
```